src/helpers/references.ts

```diff
@@ -58,8 +58,16 @@ export function isSingleCellReference(xc: string): boolean {
 }
 
 export function splitReference(ref: string): { sheetName?: string; xc: string } {
+  if (!ref.includes("!")) {
+    return { xc: ref };
+  }
   const parts = ref.split("!");
   const xc = parts.pop()!;
   const sheetName = getUnquotedSheetName(parts.join("!")) || undefined;
   return { sheetName, xc };
 }
+
+/** Return a reference SheetName!xc from the given arguments */
+export function getReference(xc: string, sheetName?: string): string {
```

Comment:
what about `getFullReferenceString` or `toFullReferenceString` ?

Reply:
Maybe `mergeReference` to be the opposite of `splitReference` ? Idk